### PR TITLE
PSR-1555 | Implement PSR-1469 Update 4

### PR DESCRIPTION
**Pull request by ccatosdevelopment**

### CODE REVIEW

File: app/controllers/nonsipp/landorproperty/LandOrPropertyLeaseDetailsController.scala [hunk 1]
```diff
@@ -163,9 +163,9 @@ object LandOrPropertyLeaseDetailsController {
       QuestionField
         .input("landOrPropertyLeaseDetails.field1.label", Some("landOrPropertyLeaseDetails.field1.label.hint")),
       QuestionField
-        .money("landOrPropertyLeaseDetails.field2.label", Some("landOrPropertyLeaseDetails.field2.label.hint")),
+        .currency("landOrPropertyLeaseDetails.field2.label", Some("landOrPropertyLeaseDetails.field2.label.hint")),
       QuestionField
-        .localDate("landOrPropertyLeaseDetails.field3.label", Some("landOrPropertyLeaseDetails.field3.label.hint"))
+        .date("landOrPropertyLeaseDetails.field3.label", Some("landOrPropertyLeaseDetails.field3.label.hint"))
```
Comment (ccatosdevelopment) [inline app/controllers/nonsipp/landorproperty/LandOrPropertyLeaseDetailsController.scala:168]: Refactoring identical methods.

File: app/viewmodels/models/MultipleQuestionsViewModel.scala [hunk 2]
```diff
@@ -134,17 +134,9 @@ object QuestionField {
...
-  def money(label: InlineMessage, hint: Option[InlineMessage] = None): QuestionField =
-    currency(label, hint)
-
   def date(label: InlineMessage, hint: Option[InlineMessage] = None): QuestionField =
     QuestionField(label, hint, None, Nil, FieldType.Date)
 
-  // alias for date
-  def localDate(label: InlineMessage, hint: Option[InlineMessage] = None): QuestionField =
-    date(label, hint)
-
```
Comment (ccatosdevelopment) [inline app/viewmodels/models/MultipleQuestionsViewModel.scala:147]: Refactoring unused identical methods.

File: test/views/IntViewSpec.scala [hunk 3]
```diff
@@ -46,16 +47,16 @@ class IntViewSpec extends ViewSpec with Mappings {
...
 
       act.like(renderTitle(viewModelGen)(view(intForm, _), _.title.key))
       act.like(renderHeading(viewModelGen)(view(intForm, _), _.heading))
       act.like(renderInputWithLabel(singleIntViewModel)("value", view(intForm, _), _.heading))
-//      act.like(renderInputWithLabel(tripleIntViewModel)("value.1", view(tripleIntForm, _), _.page.fields.head.label))
-//      act.like(renderInputWithLabel(tripleIntViewModel)("value.2", view(tripleIntForm, _), _.page.fields(1).label))
-//      act.like(renderInputWithLabel(tripleIntViewModel)("value.3", view(tripleIntForm, _), _.page.fields(2).label))
+      act.like(renderInputWithLabel(tripleIntViewModel)("value.1", view(intForm, _), _.page.fields.head.label))
+      act.like(renderInputWithLabel(tripleIntViewModel)("value.2", view(intForm, _), _.page.fields(1).label))
+      act.like(renderInputWithLabel(tripleIntViewModel)("value.3", view(intForm, _), _.page.fields(2).label))
```
Comment (ccatosdevelopment) [inline test/views/IntViewSpec.scala:59]: Same scenario as in MoneyViewSpec, see comment there.

File: test/views/MoneyViewSpec.scala [hunk 4]
```diff
@@ -48,13 +49,15 @@ class MoneyViewSpec extends ViewSpec with Mappings {
...
 
       act.like(renderTitle(viewModelGen)(view(moneyForm, _), _.title.key))
       act.like(renderHeading(viewModelGen)(view(moneyForm, _), _.heading))
-      act.like(renderInputWithLabel(singleMoneyQuestion)("value", view(moneyForm, _), _.heading))
+      act.like(renderInputWithLabel(singleMoneyQuestion)("value", view(moneyForm, _), _.page.firstField.label))
+      // TODO: remove these tests? None of the pages that use MoneyView have multiple questions, so this seems redundant
+      //  Instead, create MultipleQuestionsViewSpec, to test pages with multiple money inputs?
 //      act.like(renderInputWithLabel(tripleMoneyQuestion)("value.1", view(_), _.page.fields.head.label))
 //      act.like(renderInputWithLabel(tripleMoneyQuestion)("value.2", view(_), _.page.fields(1).label))
 //      act.like(renderInputWithLabel(tripleMoneyQuestion)("value.3", view(_), _.page.fields(2).label))
```
Comment (ccatosdevelopment) [inline test/views/MoneyViewSpec.scala:63]: In addition to what the TODO says, the same is true for for IntView. I think we should create a MoneyViewSpec to include these kinds of tests, though I'm not sure if that work should be done here or in a separate ticket.

Comment (ccatosdevelopment) [inline test/views/MoneyViewSpec.scala:63]: In addition to what the TODO says, the same is true for for IntView. I think we should create a MoneyViewSpec to include these kinds of tests, though I'm not sure if that work should be done here or in a separate ticket.

**Review by kberec — APPROVED** [review]

**Pull request merged**